doc/product-variants-migration.md

```diff
@@ -0,0 +1,25 @@
+# Product variants migration
+
+If you used variant groups and/or inner variation types (with the paid extension InnerVariationBundle for the Enterprise Edition), or both together, the product migration is done in two steps. At first the products are migrated as they are not taking variants in account. Then in a second step, family variants, product models and product variants are created in the PIM 2.0 to reproduce the variants of the PIM 1.7.
+
+However, some bad use cases of variants are not supported anymore in 2.0. Here are the rules that must be respected in order to fully migrate the product variants :
+
+- All the variants of a product must be of the same family
+- A family variant must not have more than 5 axes.
+- A variant axis must be one of the following types :
+    - Simple select
+    - Reference data simple select
+    - Metric
+    - Yes/No
+
+If a variant group or a inner variation type does not comply with these rules, the products concerned will remain without variants (you will be warned if it happens). You will have to think about a better modeling for these products and then two options are available :
```

Comment:
`the concerned products will remain without variant and you will be warned if it occurs`

Reply:
`two options are available:` -> no space before the `:`